src/components/chip-set/chip-set.tsx

```diff
@@ -513,22 +519,27 @@ export class ChipSet {
         }
 
         return (
-            <div
-                class={{
-                    'mdc-chip': true,
-                    'mdc-chip--selected': this.inputChipIndexSelected === index,
-                    disabled: this.disabled,
-                }}
-                role="row"
-                id={`${chip.id}`}
-                onClick={this.catchInputChipClicks}
-                {...attributes}
-            >
-                {chip.icon ? this.renderIcon(chip) : null}
-                {this.renderLabel(chip)}
-                {chip.removable && !this.readonly && !this.disabled
-                    ? this.renderTrailingIcon()
-                    : null}
+            <div class="chip-delimiter-wrapper">
```

Comment:
Hmm, do we need to wrap the entire chip in a new `div`? :thinking: 
It seems a bit weird that the root element of a chip is used for the delimiters when they might not even be used :slightly_smiling_face: